code/items/perks/Test1.cs

```diff
@@ -9,6 +11,33 @@ public Test1() : base()
         {
 
         }
+        public override void OnEquip()
+        {
+            base.OnEquip();
+
+            VertexBuffer vb = new VertexBuffer(); // this should be called vertex builder or something
```

Comment:
Is this comment needed, `VertexBuffer` is within s&box right? Not a change we can make.